lib/mumuki/laboratory/controllers/notifications.rb

```diff
@@ -1,6 +1,7 @@
 module Mumuki::Laboratory::Controllers::Notifications
+
   def user_notifications_path
-    "#{user_path}##{notifications_path}"
+    helpers.user_profile_path(current_notifications_tab)
   end
```

Comment:
Why `helpers.`?

Reply:
In order to avoid making it a Concern and including the helper manually, but if you think it's cleaner, i'll change it